app/src/main/java/org/jellyfin/androidtv/preference/UserPreferences.kt

```diff
@@ -200,6 +200,12 @@ class UserPreferences(context: Context) : SharedPreferenceStore(
 		 * Whether items shown in the screensaver are required to have an age rating set.
 		 */
 		var screensaverAgeRatingRequired = booleanPreference("screensaver_agerating_required", true)
+
+		/**
+		 * Preferred behavior for zoom mode.
+		 */
+
+		var zoomMode = intPreference("zoom_mode", 0)
```

Comment:
Please use an enumPreference with a (new) enum type instead. It's probably best to change the VideoManager to use the enum too.

Reply:
I have updated to use a new enum. I am still getting familiar with this language so please let me know if additional changes are required.